internal/web/templates/components/pkg-config-input.html

```diff
@@ -225,12 +225,17 @@
 {{ end }}
 
 
-<!-- this is the entry point for rendering the the input group for one value definition -->
+<!-- this is the entry point for rendering the input group for one value definition -->
 {{ define "pkg-config-input" }}
   <div id="{{ .ContainerId }}" class="mb-2">
     {{ if eq .ValueDefinition.Type "text" }}
       <div>
-        <label for="{{ .FormId }}" class="form-label">{{ .FormLabel }} (<code>{{ .ValueName }}</code>)</label>
+        <label for="{{ .FormId }}" class="form-label"
```

Comment:
Since it's the same code snippet for all types of inputs, I would recommend doing a little template for that. 

You can add something like 
```
{{ define "pkg-config-input-required" }}
 ...
{{ end }}
```

and then render this template with `{{ template "pkg-config-input-required" . }}` 4 times.

Reply:
done